addons/website_blog/views/snippets/s_latest_posts.xml

```diff
@@ -104,7 +104,7 @@
                 <we-button data-select-data-attribute="published_date desc">Latest</we-button>
                 <we-button data-select-data-attribute="visits desc">Most viewed</we-button>
             </we-select>
-            <we-select string="Layout" data-no-preview="true" data-attribute-name="template" class="o_we_inline">
```

Comment:
I think the o_we_inline class is useless now, can you remove it ?

Reply:
indeed